### ISSUE TITLE
T1 L6 S15 broken link

### ISSUE THREAD

**Issue by sangxia** [udacity/sdc-issue-reports#583]
tf.nn.softmax() points to https://www.tensorflow.org/api_docs/python/nn/classification#softmax
the new url is
https://www.tensorflow.org/api_docs/python/tf/nn/softmax

**Comment by baumanab** [comment]
Great bug catch!  The ink has been updated.
